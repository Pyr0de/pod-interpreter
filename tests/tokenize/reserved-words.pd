for
while
if let
else
func
return print
true false
for_while
