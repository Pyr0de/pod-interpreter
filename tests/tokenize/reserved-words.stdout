ExitCode: 0
FOR for <nil> 1
WHILE while <nil> 2
IF if <nil> 3
INIT let <nil> 3
ELSE else <nil> 4
FUNC func <nil> 5
RETURN return <nil> 6
PRINT print <nil> 6
TRUE true <nil> 7
FALSE false <nil> 7
IDENTIFIER for_while <nil> 8
EOF  null
